Sources/LiveViewNative/Coordinators/LiveSessionConfiguration.swift

```diff
@@ -26,17 +26,65 @@ public struct LiveSessionConfiguration {
     /// The transition used when the live view changes connects.
     public var transition: AnyTransition?
     
+    public var reconnectBehavior: ReconnectBehavior = .exponential
+    
     /// Constructs a default, empty configuration.
     public init() {
     }
     
     public init(
         connectParams: ((URL) -> [String: Any])? = nil,
         urlSessionConfiguration: URLSessionConfiguration = .default,
-        transition: AnyTransition? = nil
+        transition: AnyTransition? = nil,
+        reconnectBehavior: ReconnectBehavior = .exponential
     ) {
         self.connectParams = connectParams
         self.urlSessionConfiguration = urlSessionConfiguration
         self.transition = transition
+        self.reconnectBehavior = reconnectBehavior
+    }
+    
+    public struct ReconnectBehavior {
+        let delay: ((_ tries: Int) -> TimeInterval)?
+        
+        public init(_ delay: @escaping (_ tries: Int) -> TimeInterval) {
+            self.delay = delay
+        }
+        
+        private init() {
+            self.delay = nil
+        }
+        
+        /// Attempt to reconnect at an exponential rate.
+        public static func exponential(
+            upTo maxDelay: TimeInterval = 32,
+            step: TimeInterval = 1,
+            jitter: ClosedRange<TimeInterval> = 0...0.25
+        ) -> Self {
```

Comment:
@bcardarella Let me know if the defaults here should be tweaked.

Reply:
I think that's OK for now. We may want to make this configurable in the future.

Reply:
It already is configurable. The main issue text shows how the API works. Those are just the default values if you don't pass the configuration option.